lib/controller.dart

```diff
@@ -0,0 +1,68 @@
+library controller;
+
+import 'package:dashboard/model.dart' as model;
+import 'package:dashboard/view.dart' as view;
+import 'package:dashboard/firebase.dart' as fb;
+import 'package:dashboard/logger.dart';
+
+Logger logger = Logger('controller.dart');
+
+var _navLinks = [
+  model.Link('analyse', 'Analyse', view.renderAnalyseTab),
+  model.Link('settings', 'Settings', view.renderSettingsTab)
+];
```

Comment:
since you need to index into this list by name for `changeNavTab`, you could turn it into a map, e.g.:
```suggestion
var _navLinks = {
  'analyse': model.Link('analyse', 'Analyse', view.renderAnalyseTab),
  'settings': model.Link('settings', 'Settings', view.renderSettingsTab)
};
```

Reply:
Ah i keep forgetting i could still do map and foreach on a map, fixed